fhir-server-test/src/test/java/com/ibm/fhir/server/test/operation/VersionsOperationTest.java

```diff
@@ -0,0 +1,251 @@
+/*
+ * (C) Copyright IBM Corp. 2022
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.fhir.server.test.operation;
+
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertNotNull;
+import static org.testng.Assert.assertTrue;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import javax.ws.rs.client.WebTarget;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.Response.Status;
+
+import org.testng.annotations.Test;
+import org.w3c.dom.Document;
+import org.w3c.dom.Node;
+import org.w3c.dom.NodeList;
+
+import com.ibm.fhir.core.FHIRMediaType;
+import com.ibm.fhir.model.resource.Parameters;
+import com.ibm.fhir.model.resource.Parameters.Parameter;
+import com.ibm.fhir.model.type.Code;
+import com.ibm.fhir.server.test.FHIRServerTestBase;
+
+import jakarta.json.JsonArray;
+import jakarta.json.JsonObject;
+
+/**
+ * This class tests the $versions operation, which returns the list of supported FHIR versions,
+ * along with the default FHIR version.
+ */
+public class VersionsOperationTest extends FHIRServerTestBase {
```

Comment:
Do you see this as being useful for other tests? that is knowing the version the server supports? 

if so, maybe parts of this should be pushed down to FHIRServerTestBase

Reply:
Maybe.  Not sure at this point.  I'd say if we see a need for that in the future, to move/add common logic at that point.